modules/nlu/src/backend/index.ts

```diff
@@ -37,7 +36,13 @@ const onServerStarted = async (bp: typeof sdk) => {
     validProvidersCount,
     validLanguages
   } as NLUHealth
+}
 
+const onServerStarted = async (bp: typeof sdk) => {
+  Storage.ghostProvider = (botId?: string) => (botId ? bp.ghost.forBot(botId) : bp.ghost.forGlobal())
+  const globalConfig = (await bp.config.getModuleConfig('nlu')) as Config
```

Comment:
Why do you name this`globalConfig` as module can be configured by bots.

Reply:
Lang servers are configured globally